### PR TITLE
updating xeus

### PR DESCRIPTION
**Pull request by DerThorsten**

### CODE REVIEW

File: environment-dev.yml [hunk 1]
```diff
@@ -7,15 +7,15 @@ dependencies:
   - cxx-compiler
   # Host dependencies
   - nlohmann_json
-  - xtl
   - cppzmq
```
Comment (JohanMabille) [inline environment-dev.yml:10]: I think this one can be removed too.

**Review by JohanMabille — COMMENTED** [review]
The file `xeus-sqliteConfig.cmake.in` can be clean up a bit, removing the cppzmq dependency. I haven't checked the CMakeLists.txt file, maybe some libraries / deps can be removed too.

File: environment-dev.yml (hunk 1 above)
Comment (DerThorsten) [reply environment-dev.yml:10]: which one? xtl **is beeing removed**

Comment (anutosh491) [reply environment-dev.yml:10]: I think Johan means that cppzmq can be removed too

**Pull request merged**